### PR TITLE
wallet create fixes

### PR DESCRIPTION
**Pull request by peterpme**
1) Clicking on the created wallet correctly takes you back to the Balances tab

2) Removes triple dots menu from "Wallet Created":

image

3) Continues to show triple dots menu from "Your Account / Edit Wallets":

image

### CODE REVIEW

File: packages/app-extension/src/components/Unlocked/Settings/AddConnectWallet/index.tsx [hunk 1]
```diff
@@ -184,20 +185,23 @@ export const ConfirmCreateWallet: React.FC<{
...
           isFirst={true}
           isLast={true}
           onClick={() => {
             if (tab === TAB_BALANCES) {
-              background.request({
-                method: UI_RPC_METHOD_NAVIGATION_TO_ROOT,
-                params: [],
-              });
-            } else {
+              // Experience won't go back to TAB_BALANCES so we poke it
```
Comment (armaniferrante) [inline packages/app-extension/src/components/Unlocked/Settings/AddConnectWallet/index.tsx:193]: eli5?

**Comment by tomlinton** [comment]
Related https://github.com/coral-xyz/backpack/issues/815

File: packages/app-extension/src/components/Unlocked/Settings/AddConnectWallet/index.tsx (hunk 1 above)
Comment (peterpme) [inline packages/app-extension/src/components/Unlocked/Settings/AddConnectWallet/index.tsx:193]:
```suggestion
              // Something is preventing from working by just calling `TAB_BALANCES`. By calling 
 a different tab first and then immediately the correct one (TAB_BALANCES) we're able to get it to work
```
